freeswitch/fs/lib/astpp.cdr.php

```diff
@@ -299,6 +299,15 @@ function get_cdr_string($dataVariable, $accountid, $account_type, $actual_durati
 	$dataVariable ['calltype'] = ($dataVariable ['calltype'] == 'DID-LOCAL' || $dataVariable ['calltype'] == 'SIP-DID' || $dataVariable ['calltype'] == 'OTHER') ? "DID" : $dataVariable ['calltype'];
 	// $callerIdNumber = isset($dataVariable['effective_caller_id_number']) && !empty($dataVariable['effective_caller_id_number'])? $dataVariable['effective_caller_id_number'] :$dataVariable['caller_id'];
 	$callerIdNumber = ($dataVariable ['calltype'] == "DID") ? $dataVariable ['effective_caller_id_name'] . " <" . $dataVariable ['effective_caller_id_number'] . ">" : $dataVariable ['original_caller_id_name'] . " <" . $dataVariable ['original_caller_id_number'] . ">";
+
+        //Adding q.850 code in cdr table
+	if(table_exists('q850code', 'astpp', $logger, $db)==1) {
```

Comment:
get_cdr_string function doesn't have database object $db then how are you getting that in table_exists? Please test it out, it seems wrong.

Reply:
@smrdoshi Added missing db params, please verify.